src/test/tests_common.h

```diff
@@ -305,29 +326,98 @@ tc_sem_timedwait(sem_t *sem, int sec, int line_number)
 }
 
 void
-tc_srv_start_basic(crt_context_t *crt_ctx, pthread_t *progress_thread)
+tc_cli_start_basic(char *local_group_name, char *srv_group_name,
+		   crt_group_t **grp, d_rank_list_t **rank_list,
+		   crt_context_t *crt_ctx, pthread_t *progress_thread,
```

Comment:
we probably want to add few sanity checks to ensure things like rank_list are not null